Code/Framework/AzToolsFramework/AzToolsFramework/AssetBrowser/Views/AssetBrowserTableView.cpp

```diff
@@ -146,6 +143,20 @@ namespace AzToolsFramework
 
         void AssetBrowserTableView::OnAssetBrowserComponentReady()
         {
+            horizontalHeader()->setStretchLastSection(true);
+            setColumnWidth(0, aznumeric_cast<int>(parentWidget()->width() * 0.5f));
+            horizontalHeader()->setMinimumSectionSize(aznumeric_cast<int>(parentWidget()->width() * 0.25f));
+            horizontalHeader()->setMaximumSectionSize(aznumeric_cast<int>(parentWidget()->width() * 0.75f));
```

Comment:
suggestion: It's likely a good idea to move these magic numbers to named constants at file scope (might even want to make them configurable in future too, you never know...)